package.json

```diff
@@ -85,7 +84,7 @@
     "ember-papaparse": "0.0.3",
     "ember-pikaday": "2.2.1",
     "ember-promise-helpers": "1.0.2",
-    "ember-resolver": "^3.0.0",
+    "ember-resolver": "^2.0.3",
```

Comment:
and i just merged in resolver 4.0.0 ("greenkeeper told me so"). i click tested prior to that, seems to have worked. will rebase/resolve merge conflict and see how/if that still pans out.